woocommerce/woocommerce
# FlexSlider now uses CSS transitions for height changes instead of jQuery .animate()

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #52003.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Upload images with noticeably different heights to the same product.
2. On desktop mode, navigate to that product's page.
3. Use the arrow keys to quickly switch between the 2 images (in the video sample I've provided, I'm using the right arrow key ~10 times, you can hear the key being pressed by enabling the video sound).
4. Once you've finished pressing the arrow keys, the height change animation should NOT linger anymore.

Example of the behaviour BEFORE this PR:

https://github.com/user-attachments/assets/fc92f75b-6d2a-44ce-b69e-3cc592b9c5ae



### Changelog entry



-   [] Automatically create a changelog entry from the details below.




<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
